docs/cloud/features/single_sign_on.md

```diff
@@ -0,0 +1,151 @@
+# SSO (Single sign-on) 
+
+## Overview
+
+Tobiko Cloud supports single sign-on (SSO) through OpenID and SAML 2.0 providers. 
+
+This makes it easier to provision access to users and simplifies authentication. 
+
+
+## Setup & Prerequsites 
+
+You must have an active Tobiko Cloud instance with SSO enabled. Please contact your account team to ensure this is enabled. 
+
+If your Tobiko Cloud instance is setup to require SSO then you won't need to provide a token in your tcloud.yml configuration. 
+
+Below is an example of `tcloud.yml` configuration:
+```yaml
+projects:
+    <Project name>:
+        url: <The project URL>
+        token: <The access token>
+        gateway: <The name of the SQLMesh gateway to use with this project>
+        extras: <Optional - Any extras that should be installed with sqlmesh-enterprise>
+        pip_executable: <Optional - The path to the pip executable to use. Ex: `uv pip` or `pip3`. Must install packages to the python environment running the tcloud command>
+default_project: <The name of a project to use by default>
+```
+
+## Identity Providers
+
+Tobiko Cloud currently supports OpenID and SAML 2.0. 
+
+This provider implements [OpenID Connect Core
```

Comment:
In this section the term "This provider" is wrong since you're listing both. I think this paragraph needs to go below the OpenID header.